src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/ManagedResourceParserTests.cs

```diff
@@ -426,6 +426,19 @@ public void GenerateDesignerFileFromRtxt ([Values (false, true)] bool withLibrar
 			Directory.Delete (Path.Combine (Root, path), recursive: true);
 		}
 
+		[Test]
+		public void GenerateDesignerFileFromEmptyRtxt ()
+		{
+			var path = Path.Combine ("temp", TestName + " Some Space");
+			CreateResourceDirectory (path);
+			var task = CreateTask (path);
+			task.RTxtFile = Path.Combine (Root, path, "R.txt");
+			File.WriteAllText (task.RTxtFile, string.Empty);
```

Comment:
This test isn't testing what you think it is.  This results in a 0-byte file, and with a 0-byte input, `File.ReadLines()` returns a sequence with 0 elements, meaning we won't actually hit the error condition.

What we *actually* want is a file containing a *blank line*:

```csharp
File.WriteAllText (task.RTxtFile, Environment.NewLine);
```

*Then* `File.ReadLines()` will return a non-empty sequence, and the first (and only) value it returns will be `string.Empty`, and `string.Empty.Split(new char[]{' '}, 4)` returns `new string[]{""}` (a 1 element array), which triggers the error scenario.